microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation. Licensed under the MIT license.
+
+from assertpy.assertpy import assert_that
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.base_tools.service import Service
+from lisa.sut_orchestrator.azure.common import (
+    get_compute_client,
+    get_node_context,
+    wait_operation,
+)
+from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
+
+
+def _verify_vm_agent_running(node: Node, log: Logger) -> None:
+    service = node.tools[Service]
+    is_vm_agent_running = service.is_service_running(
+        "walinuxagent.service"
+    ) or service.is_service_running("waagent.service")
+
+    log.debug(f"verify walinuxagent or waagent running:{is_vm_agent_running}")
```

Comment:
use walinuxagent and waagent should be enough, since for freebsd, only waagent is vaild.